woocommerce/woocommerce-android
# Free Trial: Update the log in / get started screen

Summary
==========
Fix issue #8544 by updating all three Login Prologue layout files to include the `Start your free trial` button instead of the usual `Get started` one.

### Reference Figma layout

<img width="338" alt="image" src="https://user-images.githubusercontent.com/5920403/227445417-2a41343e-8452-41c3-9aff-3e1d5e629146.png">

Screenshots - Phone
==========
| Portrait  | Landscape |
| ------------- | ------------- |
| ![Screenshot_20230324_034848](https://user-images.githubusercontent.com/5920403/227446910-43cbc5d6-6b31-495c-9a6c-9b131eccca1b.png) | ![Screenshot_20230324_034839](https://user-images.githubusercontent.com/5920403/227446882-9b91b73e-eefe-4846-a4c4-2df34ea4ad7d.png) |

Screenshots - Tablet
==========
| Portrait  | Landscape |
| ------------- | ------------- |
| ![Screenshot_20230324_034953](https://user-images.githubusercontent.com/5920403/227447003-2270ed94-abdf-4688-9d53-c2e91df8f8dc.png) | ![Screenshot_20230324_035144](https://user-images.githubusercontent.com/5920403/227447093-9089e7a3-d6d4-4b93-97ca-51174967b0aa.png) |

How to Test
==========
1. Fresh install the app, making sure the `FREE_TRIAL_M2` Feature flag is activated
2. Verify that the Prologue screen is displayed following the Figma designs in both landscape and portrait mode
3. Test the same now using a Tablet

Update release notes:
